lib/ddtrace/contrib/grape/endpoint.rb

```diff
@@ -1,8 +1,8 @@
-
 require 'ddtrace/ext/http'
 require 'ddtrace/ext/errors'
 require 'ddtrace/contrib/analytics'
 require 'ddtrace/contrib/rack/ext'
+require 'ddtrace/contrib/status_code_matcher'
```

Comment:
i dont _think_ we need to include the `status_code_matcher ` file here, but in the `grape/settings.rb` file instead